packages/application-shell-connectors/src/components/custom-view-context/custom-view-context.tsx

```diff
@@ -1,11 +1,17 @@
 import { createContext, useContext, useMemo, type ReactNode } from 'react';
 import type { CustomViewData } from '@commercetools-frontend/constants';
+import {
+  type TApplicationContext,
+  useApplicationContext,
+} from '../application-context';
 
 export type TCustomViewContext = {
   hostUrl: string;
   config: CustomViewData;
```

Comment:
Nit: now that we have the data together, should we rename this to `customViewConfig`?

Reply:
Yes, that makes sense. Thanks 👍 

Updated here: fa5fa67133a501c01858a1bde3737e9387e8574d